### PR TITLE
Adding information to the README.md file

### LINKED ISSUE #14: Update the README.md
- Add the name project and description.
- Create an 'Team Members' section and link each team member name to his GitHub profile.
- Add a link to Contributing.md file.
- Create an 'Installation' section that explain how to install and run our project.

### PR DESCRIPTION
**Pull request by yaelnoyman1**
Adding information to the README.file
Resolve #14

### CODE REVIEW

**Comment by Shalevro2** [comment]
@yaelnoyman1 Please add in the first comment - Resolve #issue_id.

**Comment by avi-biton** [comment]
@yaelnoyman1 if you're referring to issue #14, please update your PR description and include `Resolve #14`

File: README.md [hunk 1]
```diff
@@ -1 +1,32 @@
...
+
+In order to run **Smarticle** you should install:
+
+- [Vagrant](https://www.vagrantup.com/downloads)
+- [VirtualBox](https://www.virtualbox.org/) hypervisor(VMM)
+
+### Run The Project
+
+1. Clone [Smarticle](https://github.com/redhat-beyond/Smarticle) repository.
+  ``` git clone https://github.com/redhat-beyond/Smarticle ``
```
Comment (rawadabu) [inline README.md]: Hey @yaelnoyman1, ``` ` ``` missing in line 29

**Comment by yaelnoyman1** [comment]
@avi-biton I updated my PR and included solution #14

**Review by Shalevro2 — APPROVED** [review]
LGTM

**Review by Anas-mhameed — APPROVED** [review]
LGTM

**Review by rawadabu — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (yaelnoyman1) [reply README.md]: Thanks rawad I fixed it

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**